src/main/java/com/google/firebase/database/DatabaseReference.java

```diff
@@ -212,7 +212,30 @@ public DatabaseReference push() {
    *
    * @param value The value to set at this location
    * @param priority The priority to set at this location
+   * @return The ApiFuture for this operation.
+   */
+  public ApiFuture<Void> setValueAsync(Object value, Object priority) {
+    return new TaskToApiFuture<>(setValue(value, priority));
+  }
+
+  /**
+   * Similar to {@link #setValueAsync(Object)} but returns a Task.
```

Comment:
For FirebaseAuth, you replicated the javadoc comments, but here you are just saying "Similar to ..."  Intentional?

Reply:
Good point. Applied the "Similar to.." convention in FirebaseAuth too.